### DISCUSSION TITLE
Daily 15/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#622] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer terminé lo de EnbaldePago, está hecho en node porque me pareció lo más fácil y rápido, hoy terminé con la edición de artículos al editar la oferta. Voy a ver si puedo hacer lo del ABM de usuarios que es lo más pesado que queda si todo ya está hecho.

**Comment by BraianTroncoso** [comment]
Ayer solucioné un bug y me asigné un par de tareas, hoy espero poder finalizarlas.
